packages/training/python/tests/integration/test_db_integration.py

```
@@ -0,0 +1,649 @@
+    def test_score_multiple_archetypes_from_database(self):
+        """Test scoring multiple archetype trajectories from database."""
+        archetypes_pnl = [
+            ("trader", 200.0),
+            ("degen", -500.0),
+            ("scammer", 300.0),
+            ("social-butterfly", 20.0),
+        ]
+        
+        scores = {}
+        for archetype, pnl in archetypes_pnl:
+            traj_data = self._insert_and_fetch_trajectory(
```

Ruff (0.14.10)

Local variable `traj_data` is assigned to but never used

Remove assignment to unused variable `traj_data`

(F841)